src/tests/ftest/io/dfuse_bash_cmd.py

```diff
@@ -20,7 +20,7 @@ class BashCmd(DfuseTestBase):
     """
 
     def test_bashcmd(self):
-        """Jira ID: DAOS-3508.
```

Comment:
No need to change or add jira ticket. Jira ID simply reflects for the first time the test was added. Modifying it every time with the ticket for which change was made will eventually make it look too clumsy and idea of an ID will no longer suffice.